### DISCUSSION TITLE
How to spawn an actor?

### DISCUSSION THREAD

**Discussion by Osten** [nxrighthere/UnrealCLR#34] [Help] [answered]
Hello!

This is probably a pretty simple question.. I am looking for `SpawnActor` in the framework and can't find it. Is this not implemented? Also, if it is, how would I have gone about finding it? I am evaluating if this is something we can use at our studio as we now move to Unreal but have experience mostly in C#. Thanks and hope to use more of this in the future!

**Comment by nxrighthere** [comment, marked as answer]
Hi there.

API of the framework written in idiomatic C# thus might differ from the original one. To spawn an actor just use a constructor of the [Actor](https://github.com/nxrighthere/UnrealCLR/blob/master/API/Actor-Actor(string_Blueprint).md) class. A good starting point to learn how things work here is to dive into [tests](https://github.com/nxrighthere/UnrealCLR/tree/master/Source/Managed/Tests) that can be installed along with the plugin.

**Reply by Osten** [reply]
much appreciated!
